pkg/reconciler/brokercell/resources/deployments.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"strconv"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/kmeta"
+	"knative.dev/pkg/system"
+)
+
+// MakeIngressDeployment creates the ingress Deployment object.
+func MakeIngressDeployment(args IngressArgs) *appsv1.Deployment {
```

Comment:
Does it complain test coverage if we don't have dedicated test?

Reply:
Yes it does... But it should have good coverage in the controller test. Any workarounds?

Reply:
I know you can skip them in `.gitattributes`. But I'm not sure if you can still see the coverage of these files and our existing resources folder are all providing dedicated tests.